tensorflow/contrib/makefile/README.md

```diff
@@ -142,10 +142,11 @@ xcode-select --install
 If this is a new install, you will need to run XCode once to agree to the
 license before continuing.
 
-Then install [automake](https://en.wikipedia.org/wiki/Automake):
+Then install [automake](https://en.wikipedia.org/wiki/Automake)/libtool:
```

Comment:
For consistency, can you make it [automake](link to automake) and [libtool](link to libtool)